README.md

```diff
@@ -61,14 +61,205 @@ target:
     #   password: "PASSWORD"#
 ~~~
 
+> Note the `repo.url` property you need to specify is the same one you would use to add the repo to helm with the `helm repo add command`.
+>
+> Example: `helm repo add bitnami https://charts.bitnami.com/bitnami`.
+
 Credentials can be provided using config file or the following environment variables:
 
 - `SOURCE_AUTH_USERNAME`
 - `SOURCE_AUTH_PASSWORD`
 - `TARGET_AUTH_USERNAME`
 - `TARGET_AUTH_PASSWORD`
 
-Current available Kinds are `HELM`, `CHARTMUSEUM` and `HARBOR`
+Current available Kinds are `HELM`, `CHARTMUSEUM` and `HARBOR`. Below you can find the compatibility matrix betweeen source and targets repositories.
+
+| Source Repo | Target Repo | Supported          |
+|-------------|-------------|--------------------|
+| HELM        | HELM        | :white_check_mark: |
+| HELM        | CHARTMUSEUM | :white_check_mark: |
+| HELM        | HARBOR      | :white_check_mark: |
+| CHARTMUSEUM | HELM        | :x:                |
+| CHARTMUSEUM | CHARTMUSEUM | :white_check_mark: |
+| CHARTMUSEUM | HARBOR      | :white_check_mark: |
+| HARBOR      | HELM        | :x:                |
+| HARBOR      | CHARTMUSEUM | :white_check_mark: |
+| HARBOR      | HARBOR      | :white_check_mark: |
+
+
+### Harbor example
+
+In the case of HARBOR kind repos, be aware that chart repository URLs are:
+
+https://$HARBOR_DOMAIN/chartrepo/$HARBOR_PROJECT
+
+So if HARBOR_DOMAIN=my.harbor.com and HARBOR_PROJECT=my-project, you would need to specify this repo in the config file like:
+
+~~~yaml
+target:
+ repo:
+   kind: HARBOR
+   url: "https://my.harbor.com/chartrepo/my-project"
+~~~
+
+## Requirements
+
+In order for this tool to be able to successfully migrate a chart from a source repository to another it must fulfill the following requirements:
+
+- If the chart has dependencies they are specified in a requirements.yaml file. Currently, if the dependencies are specified in the Chart.yaml file the tool won't be able to manage them.
```

Comment:
what's the standard to mention files? Should we use `requirements.yaml` instead?

Reply:
Let's file an issue to solve managing deps from Chart.yaml files

Reply:
Do you mean in markdown? I think they should use italics but not sure

Reply:
issue created: https://github.com/bitnami-labs/charts-syncer/issues/28